features/identity-event/org.wso2.carbon.identity.event.server.feature/resources/identity-event.properties

```diff
@@ -75,3 +75,11 @@ suspension.notification.enable=false
 suspension.notification.account.disable.delay=90
 suspension.notification.trigger.time=20:00:00
 suspension.notification.delays=30,45,60,75
+module.name.11=persistRequestObject
```

Comment:
Is this module present? Didn't we move RequestObject related event handling to one module?